unit_tests/test_events.py

```diff
@@ -338,3 +340,31 @@ def test_failed_stall_during_filter(self):
                                line="[99.80.124.204] [stdout] Mar 31 09:08:10 warning|  reactor stall 20").publish()
 
         self.assertEqual(event.severity, Severity.DEBUG)
+
+    def test_duration_format(self):
+        event = NodetoolEvent(nodetool_command="scrub --skip-corrupted drop_table_during_repair_ks_0", node='1.0.0.121',
+                              options="more options", publish_event=False)
+
+        event.duration = 2.52
+        self.assertEqual(event.duration_formatted, '2.52s')
+
+        event.duration = 326
+        self.assertEqual(event.duration_formatted, '5m26s')
+
+        event.duration = 4598
+        self.assertEqual(event.duration_formatted, '1h16m38s')
+
+        event.duration = 87400
+        self.assertEqual(event.duration_formatted, '1d0h16m40s')
```

Comment:
This is wrong approach for data driven testing.
Something like "parametrize" [1] should be used.

[1] https://docs.pytest.org/en/6.2.x/parametrize.html

Reply:
Done.

Reply:
I do not see such changes. Moreover, usage of "parameterize" was removed. Mistake?

Reply:
Now I found bug, you transform float seconds to minutes not changing 100 and 60 division difference.
2.73 input duration returns "2.73s" which is false.